third_party/lint/repos.bzl

```diff
@@ -3,23 +3,28 @@
 # SPDX-License-Identifier: Apache-2.0
 
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+load("//rules:repo.bzl", "http_archive_or_local")
 
-def lint_repos():
+def lint_repos(lowrisc_lint = None):
     # We have a 'vendored' copy of the google_verible_verilog_syntax_py repo
     native.local_repository(
         name = "google_verible_verilog_syntax_py",
         path = "hw/ip/prim/util/vendor/google_verible_verilog_syntax_py",
     )
 
+    # FIXME(cfrantz): Name collision with another package loading `com_github_bazelbuild_buildtools`?
+    # We seem to always be getting an _old_ version of buildtools that doesn't have the
+    # `buildify_test` rule.  Temporary workaround.
     http_archive(
-        name = "com_github_bazelbuild_buildtools",
-        strip_prefix = "buildtools-main",
-        url = "https://github.com/bazelbuild/buildtools/archive/main.zip",
+        name = "xcom_github_bazelbuild_buildtools",
```

Comment:
Rule 'xcom_github_bazelbuild_buildtools' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "e3bb0dc8b0274ea1aca75f1f8c0c835adbe589708ea89bf698069d0790701ea3"

Reply:
I should really find a way to make missing SHA's blocking in CI. I filed #15110 to track.

Reply:
I'd always intended to fix this before committing and its now fixed.